graphrag_sdk/agents/kg_agent.py

```diff
@@ -123,22 +124,19 @@ def kg(self, value: KnowledgeGraph):
         """
         self._kg = value
 
-    def run(
-        self, params: dict, session: GenerativeModelChatSession | None = None
-    ) -> tuple[str, GenerativeModelChatSession]:
+    def run(self, params: dict) -> tuple[str, GenerativeModelChatSession]:
```

Comment:
Incase the returned chat session isn't used. by caller, please do not return it.